issue_template.md

```diff
@@ -0,0 +1,14 @@
+### Environment
```

Comment:
Add "Task name" to the environment section?

Reply:
I didn't add it since I think the title and description are likely to contain it.